Sigma/CVE-2024-4040.yml

```diff
@@ -1,23 +0,0 @@
-title: Detects CVE-2024-4040 CrushFTP
-id: e68d5989-73b8-4320-a31b-f7453a407394
-status: experimental
-description: Detects patterns in proxy logs that might inddicare arbitrary file reads in CrushFTP.
-author: Christiaan B @ Rapid7 Labs
-logsource:
-  category: proxy
-  product: any
-  service: http
-detection:
-  selectors:
-    method:
-      - POST
-    url1|re: '/WebInterface/function/.*<INCLUDE>/[^<]+</INCLUDE>'
-  condition: any of selectors and method
-falsepositives:
-  - Legitimate administrative actions that use similar parameters or paths
-level: high
-references:
-  - https://www.akb.com/cve-2024-4040/rapid7-analysis
```

Comment:
This link does not work as analysis has not been posted yet. recommend removing for now.